QUARKUS-5477.md

```diff
@@ -0,0 +1,36 @@
+# QUARKUS-5477 Add OIDC Client SPI
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-5477
+
+Add OIDC Client SPI
+
+Upstream PRs:
+- https://github.com/quarkusio/quarkus/pull/42879
+
+Upsteram issue:
+- https://github.com/quarkusio/quarkus/issues/42878
+
+Documentation:
+- https://quarkus.io/guides/security-openid-connect-client-reference#oidc-client-spi
+
+## Scope of testing
+The `quarkus-oidc-client-spi` needs either the `quarkus-oidc-client` or `quarkus-oidc-client-deployment` dependency to be present.
+The implemented tests will validate the `TokenProvider` ability to generate access tokens for the default OIDC client.
+This implementation excludes access token provision for custom OIDC clients.
+
+
+## Automated test development
+- Test suite
+  - The `TokenProvider` tests will check:
+    - Verify successful access to a protected endpoint using a generated token.
+    - Confirm the protected endpoint returns a 401 Unauthorized HTTP status code when token expire.
+    - Validate that token is refreshed and new token can continued access to the protected endpoint.
```

Comment:
The second part of this sentence sounds strange, like it is missing a verb or something like that.

Reply:
True, I changed the wordings